index.html

```diff
@@ -3,10 +3,144 @@
+
+  <main>
+    <section class="mobile-menu" id="mobileNave">
+      <ul class="nav-list2"> 
+        <img onclick="closeNave();" class="img-close" src="./images/Icon_X_.png" alt="">
+        <li class="mobile-link" onclick="closeNave();"><a href="AboutPage.html" >About</a></li>
+        <li class="mobile-link" onclick="closeNave();"><a href="#aboutme-container" >Program</a></li>
+        <li class="mobile-link" onclick="closeNave();"><a href="#contact">News</a></li>
+    </section>
+
+    <section class="home-page" id="home-page" >
+      <div class="containerHome-Page">
+        <div>
+          <p class="greetingOne">"Hello, Sharing World!"</p>
+          <h1 class="home-title" id="home-title" >Urban Art Festival Creativity in the Streets 2023</h1>
+        </div>
+        <div class="home-Description">
+          <p>
+            "Join us at the exhilarating 'Creativity in the Streets' Urban Art Festival, where the most talented street artists come together to transform the streets of our city into an outdoor art gallery. This annual festival celebrates urban culture, artistic expression, and the creative community in a vibrant and unique atmosphere."
+          </p>
+        </div>
+        <div>
+          <p class="home-date">2023.07.03(THU) ~ 12(WEN)</p>
+          <p class="home-location-info">@StudentsToWorkd Collage-EEUU</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="activities" id="activities">
+      <div class="activities-subtitle">
+        <h2> Main Program </h2>
+        <hr>
+      </div>
+      <div class="containerBoxs">
+        <div class="boxs">
+          <img src="/images/nuevo-mensaje.png" class="imagesActivity" alt="" srcset="">
+          <h4>Forum</h4>
+          <p>Enjoy taking some time Take the time to share your thoughts and opinions with experts for each topic.</p>
+        </div>
+        <div class="boxs">
+          <img src="/images/taller.png" class="imagesActivity" alt="" srcset="">
+          <h4>Advisory</h4>
+          <p>Get free advice and recommendations from the most experts in the field, negotiate according to your interests.</p>
+        </div>
+        <div class="boxs">
+          <img src="/images/comida.png" class="imagesActivity" alt="" srcset="">
+          <h4>Aperitif</h4>
+          <p>Enjoy a delicious appetizer while you enjoy observing the greatest works of great artists of the genre.</p>
+        </div>
+        <div class="boxs">
+          <img src="/images/tendencia.png" class="imagesActivity" alt="" srcset="">
+          <h4>Trends</h4>
+          <p>Find out about the latest trends, update your knowledge, get ideas, find best and most detailed information.</p>
+        </div>
+        <div class="boxs">
+          <img src="/images/exhibicion.png" class="imagesActivity" alt="" srcset="">
+          <h4>Exhibition</h4>
+          <p>You can enjoy a wide gallery of exhibitions, where you can find a very varied range of works made in different cultures.</p>
```

Comment:
- [ ] I noticed a few elements in your main program section that don't quite match the template's style. It might be a good idea to experiment with some different text colors within your cards.
- [ ] I've observed that some of your cards vary in size, with a few appearing smaller than the rest. It could be helpful to make them consistent.
- [ ] I wanted to mention that the title below doesn't quite resemble the template. The spacing between the line and the text seems a bit too far apart. Just a small adjustment might do the trick!

   ### Your design:
   ![image](https://github.com/Fernando426/CapstoneN01/assets/107952724/1c6c3241-a895-49e7-be58-bf97d2c5233a)
 
   ### Behance design:
   ![image](https://github.com/Fernando426/CapstoneN01/assets/107952724/b14de9bb-cc9a-4fda-b2a1-0876edaea107)